### PR TITLE
[adsp] remove usage of list item entries on skins

### PR DESCRIPTION
**Pull request by AlwinEsch**
This remove the set of list entries on skin itself and do this on code side.

Further becomes with this way only the Mode type shown who is present.

Here screenshots about the changes
screenshot

screenshot1

screenshot2

### CODE REVIEW

**Comment by AlwinEsch** [comment]
@phil65 @roni @AchimTuran can check them.

**Comment by ronie** [comment]
nice, skin changes are ok.

**Comment by phil65** [comment]
Nice, thx for goin after my request that fast. :)

File: xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp [hunk 1]
```diff
@@ -704,14 +706,24 @@ void CGUIDialogAudioDSPManager::Update()
...
     modes.clear();
     db.GetModes(modes, iModeType);
 
     // No modes available, nothing to do.
     if (!modes.empty())
     {
+      CFileItemPtr item(new CFileItem());
+      item->SetLabel(g_localizeStrings.Get(dsp_mode_types[i].iName));
+      item->SetLabel2(g_localizeStrings.Get(dsp_mode_types[i].iDescription));
+      item->SetProperty("currentMode", dsp_mode_types[i].sModeType);
```
Comment (AchimTuran) [inline xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp]: Great idea @AlwinEsch, it's much better to do set this in our core codebase instead in every skin.

**Comment by AchimTuran** [comment]
I guess something similiar like `[estuary] remove no more needed adsp manager items` would also be needed for the skin confluence, because now the current mode property is set at the C/C++ side and not in the skin right?

**Comment by AlwinEsch** [comment]
Already a pull request on confluence: https://github.com/xbmc/skin.confluence/pull/9 :D

If I'm not wrong, works it also if the set is still is still in xml and the values from Kodi becomes overwritten.

**Comment by AlwinEsch** [comment]
jenkins build this please

**Pull request merged**